backend_new/src/services/translation.service.ts

```diff
@@ -0,0 +1,161 @@
+    }
+
+    const pathsToFilter = {
+      applicationPickUpAddressOfficeHours:
+        listing.applicationPickUpAddressOfficeHours,
+      costsNotIncluded: listing.costsNotIncluded,
+      creditHistory: listing.creditHistory,
+      criminalBackground: listing.criminalBackground,
+      programRules: listing.programRules,
+      rentalAssistance: listing.rentalAssistance,
+      rentalHistory: listing.rentalHistory,
+      requiredDocuments: listing.requiredDocuments,
+      specialNotes: listing.specialNotes,
+      whatToExpect: listing.whatToExpect,
+      accessibility: listing.accessibility,
+      amenities: listing.amenities,
+      neighborhood: listing.neighborhood,
+      petPolicy: listing.petPolicy,
+      servicesOffered: listing.servicesOffered,
+      smokingPolicy: listing.smokingPolicy,
+      unitAmenities: listing.unitAmenities,
+    };
+
+    listing.events?.forEach((_, index) => {
+      pathsToFilter[`events[${index}].note`] = listing.events[index].note;
+      pathsToFilter[`events[${index}].label`] = listing.events[index].label;
+    });
+
+    if (listing.listingMultiselectQuestions) {
+      listing.listingMultiselectQuestions.map((multiselectQuestion, index) => {
+        multiselectQuestion.multiselectQuestions.untranslatedText =
+          multiselectQuestion.multiselectQuestions.text;
+        multiselectQuestion.multiselectQuestions.untranslatedOptOutText =
+          multiselectQuestion.multiselectQuestions.optOutText;
+        pathsToFilter[
+          `listingMultiselectQuestions[${index}].multiselectQuestions.text`
+        ] = multiselectQuestion.multiselectQuestions.text;
+        pathsToFilter[
+          `listingMultiselectQuestions[${index}].multiselectQuestions.description`
+        ] = multiselectQuestion.multiselectQuestions.description;
+        pathsToFilter[
+          `listingMultiselectQuestions[${index}].multiselectQuestions.subText`
+        ] = multiselectQuestion.multiselectQuestions.subText;
+        pathsToFilter[
+          `listingMultiselectQuestions[${index}].multiselectQuestions.optOutText`
+        ] = multiselectQuestion.multiselectQuestions.optOutText;
+        // TODO: should we translate links?
+        multiselectQuestion.multiselectQuestions.options?.map(
+          (multiselectOption, optionIndex) => {
+            multiselectOption.untranslatedText = multiselectOption.text;
+            pathsToFilter[
+              `listingMultiselectQuestions[${index}].multiselectQuestions.options[${optionIndex}].text`
+            ] = multiselectOption.text;
+            pathsToFilter[
+              `listingMultiselectQuestions[${index}].multiselectQuestions.options[${optionIndex}].description`
+            ] = multiselectOption.description;
+            pathsToFilter[
+              `listingMultiselectQuestions[${index}].multiselectQuestions.options[${optionIndex}].description`
+            ] = multiselectOption.description;
+          },
+          // TODO: should we translate links?
```

Comment:
Is this TODO still needed?

Reply:
It is a question for the team, it is being done in the existing translations service however I don't think it is needed. In HBA there are no multiselect questions with links or options with links. I would also assume that links shouldn't be translated in general

Reply:
@YazeedLoonat do you have an opinion on this?

Reply:
if we don't have any right now I think we're safe to remove these and if we end up having a link that needs to get translated we can do that then 

like you said I don't think we want to translate links at all